apps/emqx_connector/src/emqx_connector_api.erl

```diff
@@ -795,6 +795,8 @@ maybe_unwrap(RpcMulticallResult) ->
 redact(Term) ->
     emqx_utils:redact(Term).
 
+map_to_json({bad_ssl_config, Map}) when is_map(Map) ->
+    map_to_json(Map);
```

Comment:
I'd argue this is a weird place for that, given that `emqx_connector` already tries to fit [all other validation-ish errors](https://github.com/emqx/emqx/blob/c17c7884380313d1b2075da90fa9ec1cada9dc48/apps/emqx_connector/src/emqx_connector.erl#L541-L545) into a common format. So one would naturally expect this is taken care of there as well.

Reply:
Makes sense.  Adapting the [originating error](https://github.com/emqx/emqx/blob/c17c7884380313d1b2075da90fa9ec1cada9dc48/apps/emqx_connector/src/emqx_connector.erl#L355) might be better, indeed.